src/packages/core/tests/server.test.ts

```diff
@@ -62,6 +62,49 @@ describe("server", () => {
       await s.close();
     }
   }
+  /**
+   * Sends a post request to the server and returns the response.
+   * @param address
+   * @param port
+   * @param json
+   * @param agent
+   * @returns
+   */
+  function post(host: string, port: number, json: any, agent?: any) {
```

Comment:
Is this `post` and the other `post` below still similar enough that we can combine them into one function?

Reply:
Yeah my bad - I have removed the duplicate.